front/app/containers/Admin/formBuilder/components/FormBuilderSettings/LogicSettings/index.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+
+// components
+import { Box, colors } from '@citizenlab/cl2-component-library';
+import Warning from 'components/UI/Warning';
+import { RuleInput } from './RuleInput';
+
+// intl
+import messages from '../../messages';
+import { useIntl } from 'utils/cl-intl';
+
+// services & hooks
+import { IFlatCustomFieldWithIndex } from 'services/formCustomFields';
+import useLocale from 'hooks/useLocale';
+import { isNilOrError } from 'utils/helperUtils';
+import { useFormContext } from 'react-hook-form';
+
+type LogicSettingsProps = {
+  pageOptions: { value: string; label: string }[];
+  field: IFlatCustomFieldWithIndex;
+};
+export const LogicSettings = ({ pageOptions, field }: LogicSettingsProps) => {
+  const { formatMessage } = useIntl();
+  const { watch } = useFormContext();
+  const locale = useLocale();
+  const selectOptions = watch(`customFields.${field.index}.options`);
+  const linearScaleMaximum = watch(`customFields.${field.index}.maximum`);
+
+  if (isNilOrError(locale)) {
+    return null;
+  } else {
+    // Select Field
+    let answers = selectOptions
```

Comment:
Minor: Let's add a type for answers here. It will help us track if something shouldn't be